### ISSUE TITLE
.sshyncignore's bug fix

### ISSUE THREAD

**Issue by kz-inada** [mateogianolio/sshync#25]
```
-----------------------------------------------
     );
 
   var source = path.resolve(args[0]),
-      exclude = path.relative(source, '.sshyncignore'),
+      exclude = path.resolve(source, '.sshyncignore'),
       cmd = new rsync()
         .shell('ssh')
         .flags('avuz')
-----------------------------------------------
       handle;
 
   if (fs.existsSync(exclude))
-    cmd.set('exclude-from', path.relative(source,'.sshyncignore'))
+    cmd.set('exclude-from', exclude)
 
   // abort rsync on process exit
   function quit() {
-----------------------------------------------
```

**Comment by tracyhenry** [comment]
Hi, @mateogianolio  

Currently `.sshyncignore` doesn't work at all. And this is a perfect fix for me. Is there a reason that this shouldn't be merged?

**Comment by mateogianolio** [comment]
Sorry I have just missed this, fixed in 1.1.2.

**Comment by kz-inada** [comment]
Thanx!
